### ISSUE TITLE
Missing documentation of hidewindow configuration

### ISSUE THREAD

**Issue by buddhikac96** [winsw/winsw#657]
Documentation for `hidewindow` configuration is missing.

## Suggested solution

* Add documentation to XML and YAML config documentation
* Update the YAML schema so that it includes documentation for the field

**Comment by FireCubeStudios** [comment]
hello can i try this

**Comment by buddhikac96** [comment]
@FireCubeStudios Sure! Your contribution is very welcomed!

**Comment by FireCubeStudios** [comment]
sorry i am unable to do this. been a really busy month :/

**Comment by leo-oliveira2** [comment]
Hello, can I do this?

**Comment by meiranzheng** [comment]
Opened PR #1122 to document the <hidewindow> option in the XML config docs (and add a sample snippet). I couldn't find YAML config docs/schema on the v3 branch; happy to follow up if there’s a preferred location.

**Comment by meiranzheng** [comment]
This should be resolved by #1122 (merged). If everything looks good on your side, can we close this issue?

**Comment by oleg-nenashev** [comment]
Agreed , we can close it. Normally I close the issues after the release, but the documentation is currently served from the branch and doesn't require a release

**Comment by oleg-nenashev** [comment]
Agreed , we can close it. Normally I close the issues after the release, but the documentation is currently served from the branch and doesn't require a release
